lib/join/server_boundkeypair.go

```diff
@@ -69,7 +69,13 @@ func (s *Server) handleBoundKeypairJoin(
 		return rotationResp, trace.Wrap(err)
 	}
 	generateBotCerts := func(ctx context.Context, previousBotInstanceID string, claims any) (*messages.Certificates, string, error) {
-		botCertsParams, err := makeBotCertsParams(diag, authCtx, boundKeypairInit.ClientParams.BotParams, claims)
+		botCertsParams, err := makeBotCertsParams(
+			diag,
+			authCtx,
+			boundKeypairInit.ClientParams.BotParams,
+			claims,
+			nil, // TODO(timothyb89): workload id claims
```

Comment:
@timothyb89 this TODO was in the original bound keypair implementation, copied here now that one of the join methods actually has workload ID claims and it became a parameter here

Reply:
Thanks for the heads up - this one isn't urgent so I see no issues copying the comment.